flink-ml-iteration/pom.xml

```diff
@@ -61,6 +61,14 @@ under the License.
             <scope>provided</scope>
         </dependency>
 
+        <!-- We have special treatment with the rocksdb state. -->
```

Comment:
I think we might not since in the main code we have references to rocksdb related classes. With provided, the dependency would also not be included into the jar.